cmake/zycore-config.cmake.in

```diff
@@ -7,6 +7,8 @@ if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux" AND NOT @ZYAN_NO_LIBC@)
     find_dependency(Threads)
 endif()
 
+include("${CMAKE_CURRENT_LIST_DIR}/zyanFunctions.cmake")
```

Comment:
Nit: can we rename this file to `zyan-functions.cmake` to be consistent with the existing file in this directory?

Reply:
Ops, didn't even thought about that, so used to camelCase names in CMake... Done